js-guide.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Newbie Guide: JavaScript</title>
+  </head>
+  <body>
+    <h1>Newbie Guide: JavaScript</h1>
+    <h2>Brief history of language</h2>
+    <p>JavaScript is an interpreted programming language originally created in 1995 by Brendan Eich. It is known as the language of the web, but it can also be run outside of a web browser.</p>
```

Comment:
Maybe include a short anecdote of how Eich created it in under a week? It's an interesting story, especially when we see how far the language and its uses have come in the last 20+ years.